application/src/main/java/org/togetherjava/tjbot/commands/AbstractCommand.java

```diff
@@ -1,18 +1,28 @@
-package org.togetherjava.tjbot.commands.example;
+package org.togetherjava.tjbot.commands;
 
 import org.jetbrains.annotations.NotNull;
-import org.togetherjava.tjbot.commands.Command;
 
 /**
  * Makes usage of constructor to store the {@link #commandName}, {@link #description} and the
  * {@link #isGuildOnly} <br>
  * This helps against a tiny bit of duplicated code.
+ * 
+ * @see Command
```

Comment:
Link this properly. `{@link Command}`.

Reply:
So `@see {@link Command}`?
Pretty sure it works like this too, so what's the reason for the change?

Reply:
You are right Tais, `@see Command` is correct if you imported `Command` or it is in the same package (see https://docs.oracle.com/en/java/javase/17/docs/specs/javadoc/doc-comment-spec.html).